packages/constants/src/permissions.ts

```diff
@@ -0,0 +1,94 @@
+import { GENERAL_IMPORTABLE_DATA_TYPES } from './importable';
+
+const REFERENCE_TYPES_NOUNS = GENERAL_IMPORTABLE_DATA_TYPES.map(
```

Comment:
I think this misses some, like `ProgramRegistryConditionCategory` (which is in `PROGRAM_REGISTRY_REFERENCE_TYPES` rather than `GENERAL_IMPORTABLE_TYPES`)